storage/remote/queue_manager.go

```diff
@@ -20,13 +20,20 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/common/log"
 	"github.com/prometheus/common/model"
+	"github.com/prometheus/prometheus/config"
+	"github.com/prometheus/prometheus/relabel"
 )
 
 // String constants for instrumentation.
 const (
 	namespace = "prometheus"
 	subsystem = "remote_storage"
 	queue     = "queue"
+
+	defaultQueueCapacity     = 100 * 1024 / 10
```

Comment:
Description for choice of value would help. I suppose each number here stands for something particular but I don't know what.